### ISSUE TITLE
Gradient in @background-color-highlight throws an error

### ISSUE THREAD

**Issue by simurai** [fv0/native-ui#74]
When Atom packages try to use the `@background-color-highlight` variable with `darken()`, `lighten()`, `mix()` etc., they get an error. I guess it's because the gradient used here

https://github.com/fv0/native-ui/blob/1502430275dfb882067741c46d601d448e795288/styles/ui-variables.less#L42

isn't a "valid" color?

### Steps to reproduce

1. Use `native-ui` theme.
2. Install https://atom.io/packages/atom-ide-ui

Error shown:

```
Error compiling Less stylesheet: /Users/simurai/.atom/packages/atom-ide-ui/node_modules/atom-ide-ui/pkg/atom-ide-datatip/styles/datatip.less

Line number: 28
error evaluating function `mix`: color2.toHSL is not a function
```

Would it be possible to use a "flat" color for the `@background-color-highlight` variable? Might not be perfect since this theme tries to imitate macOS.

**Comment by fv0** [comment]
@simurai: Thanks for mentioning this. I replaced `@background-color-highlight` with a flat color and I’m now using a different variable for the treeview to keep the `linear-gradient`. Could you update to 0.22.1 and close this issue if this fixes it? I installed atom-ide-ui and had no errors.

**Comment by jasonrudolph** [comment]
0.22.1 fixes the issue for me. Thanks for the lightning fast turnaround! :bow:

**Comment by simurai** [comment]
Yes, it's fixed in `0.22.1`. 👍
